api/desecapi/metrics.py

```diff
@@ -39,5 +39,8 @@ def set(name, *args, **kwargs):
 #pdns_change_tracker.py metrics
 
 set('desecapi_pdns_catalog_updated', 'number of times pdns catalog was updated successfully')
-#.py metrics
+
+#mail_backends.py metrics
+
+set('desecapi_sent_messages_count', 'number of messages sent')
```

Comment:
`_count` is not a good label. Let's use `desecapi_messages_sent`, and make it a histogram. We can discuss how to do that (as the structure currently is set up for counters only)

Reply:
should we use function overloading?

Reply:
As we are renaming `set` to `set_counter` anyways, we can also create another function called `set_histogram` that initializes a `Histogram` and adds it to the `metrics` dictionary.